### PR TITLE
create soy template fast_travel_between_stops

### PR DESCRIPTION
**Pull request by lilian-hunt**
TODO: create tests

### CODE REVIEW

**Review by jupi3200 — CHANGES_REQUESTED** [review]
Can you please add a screenshot and unit test?

**Review by aannie-y — COMMENTED** [review]
lgtm

**Comment by lilian-hunt** [comment]
A test has been created
Screen Shot 2021-02-10 at 2 15 01 pm

File: webapp/src/main/webapp/test/output.test.js [hunk 1]
```diff
@@ -82,6 +82,54 @@ describe('Output', function() {
...
+<th>Travel Speed km/h</th>\
+<th>Stop Sequence</th>\
+</tr>\
+</thead>\
+<tbody>\
+<tr>\
+<td>trip1</td>\
+<td>4000</td>\
+<td>6,7</td>\
+</tr>\
```
Comment (jupi3200) [inline webapp/src/main/webapp/test/output.test.js]: Nit: To help improve readability, can you possibly have the multiple columns of a single row in a single line?

Comment (lilian-hunt) [reply webapp/src/main/webapp/test/output.test.js]: Sure

Comment (lilian-hunt) [reply webapp/src/main/webapp/test/output.test.js]: Done now all current PRs

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**